arch/x86/Makefile

```
@@ -81,6 +81,7 @@ ifeq ($(CONFIG_X86_KERNEL_IBT),y)
 #   https://gcc.gnu.org/bugzilla/show_bug.cgi?id=104816
 #
 KBUILD_CFLAGS += $(call cc-option,-fcf-protection=branch -fno-jump-tables)
+KBUILD_RUSTFLAGS += -Z cf-protection=branch -Z no-jump-tables
```

We have been using the no-space convention:

```suggestion
KBUILD_RUSTFLAGS += -Zcf-protection=branch -Zno-jump-tables
```